### ISSUE TITLE
[a11y] Pressing menu does not read contents of Send Usage Data, Clear all Cookies settings screens

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#2925]
## Steps to reproduce
- enable voiceview
- Enter Send Usage Data or Clear all Cookies settings screens
- (nothing is read: bug #2924)
- Press menu

### Expected behavior
Screen contents are re-read

### Actual behavior
Nothing is announced.

### Device information
* Fire TV device: ?
* Latest affected Firefox version: v4.5 master 2cc87063e1808eaaa291b818cb22c952c9469607
* Earliest affected Firefox version: ?
* Turbo Mode: ?
